src/components/data_search/DatasetSearchTableConstants.tsx

```diff
@@ -127,20 +125,6 @@ export const makeStudyTableHeaders = (datasets: DatasetTerm[], selected: number[
         label: datasets[0].study.studyName
       })
     },
-    {
-      label: 'Description',
-      sortable: true,
-      cellStyle: makeHeaderStyle(studyCellWidths.description),
-      cellDataFn: (datasets) => ({
```

Comment:
wasn't this exact change made in this PR too? https://github.com/DataBiosphere/duos-ui/pull/2756

Reply:
Oh, I think it may be because I messed up the git, lemme try to fix that.